packages/edit-site/src/store/test/actions.js

```diff
@@ -42,17 +42,6 @@ function createRegistryWithStores() {
 }
 
 describe( 'actions', () => {
-	describe( 'toggleFeature', () => {
```

Comment:
Maybe it's worth keeping a unit test for `toggleFeature` around? I'm not sure. On the one hand, it would be a very dumb test that barely tests anything. On the other hand, we can't ever delete `toggleFeature` and so it's all that we would have to ensure that there aren't any typos or syntax errors or whatever inside `toggleFeature`.

Reply:
I think I was just happy to delete stuff. 😄 

I'll add a commit that brings it back.

Reply:
Done in [94eef52](https://github.com/WordPress/gutenberg/pull/39158/commits/94eef522c8115ebd6a5358491c3f9ec32044b12b).

I've only added back the action test as its easier to test a registry selector there, but I've expanded the test so that it does more testing of the selector.